scripts/kafka_log_stream_generator.py

```diff
@@ -0,0 +1,195 @@
+# Parseable Server (C) 2022 - 2024 Parseable, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import sys
```

Comment:
All tests can be added to https://github.com/parseablehq/quest

Reply:
Quest would be a good place for centralized testing. However, this requires:

- Converting Python to Go code
- Evaluating Go Kafka clients:
  * [Sarama](https://github.com/Shopify/sarama): Most popular, feature-rich but complex API
  * [kafka-go](https://github.com/segmentio/kafka-go): Clean API, native Go but fewer features
  * [confluent-kafka-go](https://github.com/confluentinc/confluent-kafka-go): librdkafka bindings, production-ready but has C deps
- Integration with Quest test framework
- Understanding Quest codebase

If it's not blocking, suggest tracking Quest migration in separate ticket. Would like to focus on fixing integration test OOM issues before the new year break to keep current work stable.